### PR TITLE
Issue#101

### PR DESCRIPTION
**Pull request by LucieContamin**
Resolve issue #101

### CODE REVIEW

File: R/base.R [hunk 1]
```diff
@@ -80,15 +80,15 @@ print.experiment <- function(x, interspace = 3, n = 6, digits = 4,
   attrs <- attributes(x)
 
   print_info <- function() {
-    cat(  "experiment name:    ", attrs$experiment,
-          "\ninput gaml file:    ", attrs$model,
-          "\noutput directory:   ", attrs$wkdir, "\n")
+    cat(  "experiment name: ", attrs$experiment,
```
Comment (choisy) [inline R/base.R]: Please delete the space after ":" for these two lines too.

File: R/base.R [hunk 2]
```diff
@@ -80,15 +80,15 @@ print.experiment <- function(x, interspace = 3, n = 6, digits = 4,
...
+          "\ninput gaml file: ", attrs$model,
```
Comment (choisy) [inline R/base.R]: Please remove space after ":"

File: R/call_gama.R [hunk 3]
```diff
@@ -42,7 +42,7 @@ call_gama <- function(parameter_xml_file, hpc, output_dir = "") {
   if (!dir.exists(output_dir))
     dir.create(output_dir, recursive = TRUE)
 
-  cat(paste0("Running experiment plan ... \n"))
+  cat("Running experiment plan ... \n")
```
Comment (choisy) [inline R/call_gama.R]: Please remove space before "..."

Comment (LucieContamin) [reply R/call_gama.R]: Done

**Comment by LucieContamin** [comment]
I make the change (remove space before "...") but I keep the space after the ":" in the message to keep the alignment between each line.

**Pull request merged**